src/Common/src/System/Net/Mail/MailAddressParser.cs

```diff
@@ -309,7 +309,18 @@ private static string ParseDisplayName(string data, ref int index, bool expectMu
 
         internal static string NormalizeOrThrow(string input)
         {
+#if NETSTANDARD1_7
+            try
+            {
+                return input.Normalize(Text.NormalizationForm.FormC);
+            }
+            catch (ArgumentException e)
+            {
+                throw new FormatException(SR.MailAddressInvalidFormat, e);
+            }
+#else
```

Comment:
Was getting compiler error that these apis were not found. Seems like they were failing for netcore50 configuration which had different project.json. Will revert this.